RWrapper/mspc_1.0.1/DESCRIPTION

```diff
@@ -0,0 +1,19 @@
+Package: mspc
+Title: MSPC program
+Version: 1.0.0
+Authors@R: 
```

Comment:
Authors need to be added (you of course, plus me and Vahid)

Reply:
Yes of course, i will add them.